k8s/base/1_configmap.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 data:
-  collector.yaml: |
+  collector.yaml: |-
```

Comment:
Could you confirm if this change generated or manually added ?

Reply:
yes it was autogenerated